examples/rl/simple_example.ipynb

```diff
@@ -323,7 +323,7 @@
     "simulator = SimpleSimulator(100.0, NSTEPS)\n",
     "state = simulator.get_state()\n",
     "obs = [{\"obs\": state_interpreter.interpret(state)}]\n",
-    "policy_out = policy(Batch(obs))\n",
+    "policy_out = policy(Batch(obs, info=None))\n",
```

Comment:
This was a very long time ago, since then all the RL examples save for this notebook have been deleted from the repository, and it may have been fixed since. I recall I had the latest tianshou version at the time (0.4.11) and just ran the script before realizing this issue.